### ISSUE TITLE
Thumbnail breaks using gw2.easy-ipfs.com Gateway (Pro.NFTMaker.io Gateway)

### ISSUE THREAD

**Issue by CardcoinsUniverse** [SmaugPool/pool.pm#11]
I just minted some NFTs using Pro.NFTMaker.io and it is currently using gw2.easy-ipfs.com as the Gateway for the IPFS Hash.

Those NFTs does not have a Thumbnail on Pool.pm when we view them in a wallet:
https://pool.pm/addr1q8psss868573q4jxpa9lyapsg26rt9k3l05acuak4yr0he24a2c7fp99npafecz638q2dz8uwz2fdjvet5j4af7e9mfsjp84fk

But when we enter the Token page on Pool.pm, we can see it with not problem:
https://pool.pm/b4e2f329afc95052d4cfb3bf60cee399e7ad9550012b308704618e50.Cardcoin10001

This is the NFT IPFS Hash: 
QmRtM17k8aBwFhES6MRpeLpXkoM6T6XKhSnziWE8EQqMQ7/10001.png

This is the Gateway link generated by Pro.NFT-Maker.io:
https://gw2.easy-ipfs.com/ipfs/QmRtM17k8aBwFhES6MRpeLpXkoM6T6XKhSnziWE8EQqMQ7/10001.png

If I Inspect any Token on Pool.pm and change only the IPFS Hash for my Hash, I get to see the Thumbnail, for example using Infura:
https://infura-ipfs.io/ipfs/QmRtM17k8aBwFhES6MRpeLpXkoM6T6XKhSnziWE8EQqMQ7/10001.png
![image](https://user-images.githubusercontent.com/91651908/136674115-1e7ce43e-7f60-4867-8a16-66d0d8e59f98.png)

But if I try to replace the Thumbnail link for the one Pro.NFT-Maker.io is using, the Thumbnail breaks:
https://gw2.easy-ipfs.com/ipfs/QmRtM17k8aBwFhES6MRpeLpXkoM6T6XKhSnziWE8EQqMQ7/10001.png
![image](https://user-images.githubusercontent.com/91651908/136674117-ce4101c5-9051-4029-a7d0-8189ed0bce3d.png)

So, basically:
gw2.easy-ipfs.com Gateway is not working fine on Pool.pm, at least not on the Thumbnails.

Is that a problem on pool.pm side? pro.nft-maker.io? Gw2 Easy IPFS?

**Comment by SmaugPool** [comment]
This token metadata does not have an `image` property and is therefore not compliant to CIP 25:

https://cips.cardano.org/cips/cip25/

As the `image` property is used as the thumbnail, this token does not have one.

**Comment by CardcoinsUniverse** [comment]
Why would Infura works as a thumbnail tho?
